far/FarEng.hlf.m4

```diff
@@ -4210,12 +4210,15 @@ be separated with a character "#;#" or "#,#". If the name of a target contains
 the character ";" or ",", it must be enclosed in quotes.
 
     If you wish to create the destination folder before copying, terminate the
-name with backslash. Also in the Copy dialog you can press #F10# to select a
-folder from the active file panel tree or #Alt+F10# to select from the passive
-file panel tree. #Shift+F10# allows to open the tree for the path entered in
-the input line (if several paths are entered, only the first one is taken into
-account). If the option "Process multiple destinations" is enabled, the dialog
-selected in the tree is appended to the edit line.
+name with backslash.
+
+    In the Copy dialog, you can press #F10# to select a folder from the
+active file panel tree or #Alt+F10# to select from the passive file
+panel tree. #Shift+F10# allows to open the tree for the path entered
+in the input line (if several paths are entered, only the first one is
+taken into account). If the option "#Process multiple destinations#"
+is enabled, the dialog selected in the tree is appended to the edit
```

Comment:
Sure, I'll fix it. I did not translate this paragraph, just split it into two because the topics are unrelated. (Did the same in Russian help.) I was going to rewrite it, but was not sure about the intended behavior of `Shift+F10`. It looks like something does not work around this place. I asked the question at the [forum](https://forum.farmanager.com/viewtopic.php?p=148538#p148538).